cli/popper/gha.py

```diff
@@ -218,7 +218,8 @@ def get_workflow_env(wf, workspace):
 
     def run(self, action, skip_clone, skip_pull, skip, workspace,
             reuse, dry_run, parallel, with_dependencies, engine,
-            engine_conf, skip_secrets_prompt=False):
+            engine_conf, substitutions, allow_loose,
```

Comment:
these arguments are not used in this function, so there's no need to pass them

Reply:
Actually they can't be removed because from cmd_run file kwargs are being passed to run function and hence If I don't create a placeholder here it throws some error.

Reply:
Actually they can't be removed directly because the kwargs are being passed to this run function from cmd_run and due to which I created a placeholder in this function. To remove from here, I can try other methods like creating a temporary tuple and then putting all the parameters except these 2 and then modifying kwargs.

Reply:
they need to be removed from the `kwargs` dictionary. I pushed this small change

Reply:
Ok Thanks a lot, From where I studied it was told that tuple and hence I thought we cannot modify it. I got to learn a new thing.